src/http/oauth1.ts

```diff
@@ -2,13 +2,12 @@ import axios from 'axios';
 import OAuth from 'oauth-1.0a';
 import { HmacSHA1 } from 'crypto-es/lib/sha1';
 import { Base64 } from 'crypto-es/lib/enc-base64';
-import { ParamsOauth1 } from '../types';
+import { Oauth1AuthParams, ParamsOauth1 } from '../types';
```

Comment:
Would `oAuth1AuthenticateParams` and `Oauth1ClientParams` make more sense here? Seeing those next to each other is really weird.

If we even can change the names without breaking backwards compatibility

Reply:
The same is true for oAuth2 file